yarn-project/aztec-nr/aztec/src/note/note_getter_options.nr

```diff
@@ -53,6 +53,17 @@ impl Sort {
     }
 }
 
+struct NoteStatusEnum {
+    ACTIVE: u2,
+    ACTIVE_OR_NULLIFIED: u2,
+}
+
+global NoteStatus = NoteStatusEnum {
+    ACTIVE: 1,
+    ACTIVE_OR_NULLIFIED: 2,
+    // TODO 4208: add 'NULLIFIED'
```

Comment:
#4208 is your old PR, should this point to an #4217? 👀

Reply:
You're right of course :sweat_smile: